src/Core/Services/Implementations/FeatureRoutedCacheService.cs

```diff
@@ -137,9 +137,11 @@ public async Task BaseDeleteOrganizationAbilityAsync(Guid organizationId)
         }
         else
         {
+            // NOTE: This is a temporary workaround InMemoryServiceBusApplicationCacheService legacy implementation.
+            // Avoid using this approach in new code.
             if (inMemoryApplicationCacheService is InMemoryServiceBusApplicationCacheService serviceBusCache)
             {
-                await serviceBusCache.DeleteOrganizationAbilityAsync(organizationId);
+                await serviceBusCache.BaseDeleteOrganizationAbilityAsync(organizationId);
```

Comment:
What is the preferred approach in new code?

Reply:
Addressed this in the other [thread’s comment](https://github.com/bitwarden/server/pull/6170#discussion_r2337595609).